torchaudio/_extension.py

```diff
@@ -125,7 +125,12 @@ def _init_extension():
 
 
 def _check_cuda_version():
-    version = torch.ops.torchaudio.cuda_version()
+    try:
+        version = torch.ops.torchaudio.cuda_version()
+    except Exception:
+        _LG.debug("Could not determine CUDA version of TorchAudio build; skipping CUDA version check.", exc_info=True)
+        return None
```

Comment:
Looking at [the failure like this](https://github.com/pytorch/audio/pull/2921#issuecomment-1370994739), it would be better to specify the condition we allow this to fail. Can we do

```python
if not _get_lib_path("libtorchaudio").exists():
    return None
version = torch.ops.torchaudio.cuda_version()
```